src/container/srv_container.c

```diff
@@ -1295,10 +1295,18 @@ cont_destroy_bcast(crt_context_t ctx, struct cont_svc *svc,
 
 	out = crt_reply_get(rpc);
 	rc = out->tdo_rc;
-	if (rc != 0) {
-		D_ERROR(DF_CONT": failed to destroy %d targets\n",
-			DP_CONT(svc->cs_pool_uuid, cont_uuid), rc);
-		rc = -DER_IO;
+	if (rc == -DER_BUSY) {
```

Comment:
in some cases, if the container is still in use (like open handle still exists with the user), a DER_BUSY error returned to the user is actually what is desired.
is this case going to be different now because of this change? are we going to endlessly retry till the container handle is closed everywhere (which is not desirable).

apologies if i misunderstood.

Reply:
@mchaarawi, I'd like to hear your thoughts before requesting landing. Does my previous reply address your concern?

Reply:
yes it's fine for me if the busy is returned before here for open handles.